speedcomplainer.py

```diff
@@ -101,8 +108,9 @@ def run(self):
         self.tweetResults(speedTestResults)
 
     def doSpeedTest(self):
-        # run a speed test
-        result = os.popen("/usr/local/bin/speedtest-cli --simple").read()
+        # Find speedtest-cli and run test
+        cli_path = subprocess.check_output('which speedtest-cli', shell=True)

```

Comment:
Could you not add _.rstrip()_ to the end of L112 and then not need to do the _.replace('\n','')_ on L113?